### PR TITLE
PHPCS for TextArea and Datepicker

### PR DESCRIPTION
**Pull request by dlh01**
See #271.

### CODE REVIEW

File: php/class-fieldmanager-datepicker.php [hunk 1]
```diff
@@ -93,23 +101,26 @@ public function form_element( $value ) {
...
+	 * @param mixed $current_value The current value for the field.
+	 * @return int Unix timestamp.
 	 */
 	public function presave( $value, $current_value = array() ) {
 		$time_to_parse = sanitize_text_field( $value['date'] );
 		if ( isset( $value['hour'] ) && is_numeric( $value['hour'] ) && $this->use_time ) {
 			$hour = intval( $value['hour'] );
 			$minute = ( isset( $value['minute'] ) && is_numeric( $value['minute'] ) ) ? intval( $value['minute'] ) : 0;
-			if ( $hour == 0 && $this->use_am_pm ) $hour = 12;
+			if ( 0 == $hour && $this->use_am_pm ) {
```
Comment (kjbenk) [inline php/class-fieldmanager-datepicker.php]: Should this use the strict comparison?

**Comment by kjbenk** [comment]
Very minor comment, otherwise :golf:

File: php/class-fieldmanager-datepicker.php (hunk 1 above)
Comment (dlh01) [inline php/class-fieldmanager-datepicker.php]: Possibly, but I don't want to risk changing the field's behavior in this PR, which is intended only to make the field standards-compliant.

Comment (kjbenk) [inline php/class-fieldmanager-datepicker.php]: Ok, but I thought using a strict comparison was making it standards compliant :)  Though it is only a warning so I guess it is fine.

Comment (mboynes) [inline php/class-fieldmanager-datepicker.php]: I don't see a problem with making that change now

**Pull request merged**